### PR TITLE
docs(ipfs): edit quickstart & api/cli section MTA-3412

### PR DESCRIPTION
**Pull request by farhakennar**
https://jira.infra.online.net/projects/MTA/issues/MTA-3412?filter=allissues

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+In this quickstart, we show you how you can [create and manage your volumes](/labs/ipfs/api-cli/volumes-operations) and [pins](/labs/ipfs/api-cli/ipfs-operations) from the [console](https://console.scaleway.com/login), via the [Scaleway Command Line Interface](https://github.com/scaleway/scaleway-cli).
+
+<Message type="important">
+  - Data is **public**: our IPFS nodes are bootstrapped with public IPFS nodes. This implies that any pinned content will be **available on the public IPFS network**.
+  - Data is **shared**: public IPFS nodes can fetch and host your pinned content. This means your data could be hosted **anywhere** from America, Asia and everything in between. Once the content is delivered to external peers, Scaleway cannot delete it.
+  - Data is **not encrypted**: we **do not apply any encryption** algorithm to your data.
+</Message>
+
+## Scaleway Command Line Interface Overview
```
Comment (bene2k1) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:34]:
```suggestion
## Scaleway Command Line Interface overview
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  posted: 2023-07-07
+categories:
+  - storage
+  - labs
+  - ipfs-storage
+  - ipfs-pinning
+  - pinning
+---
+
+Created by [Protocol Labs](https://protocol.ai/), InterPlanetary File System (IPFS) is a decentralized protocol used to store and share content.
```
Comment (bene2k1) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:20]: maybe put the link at the end for seo

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+<Message type="important">
+  - Data is **public**: our IPFS nodes are bootstrapped with public IPFS nodes. This implies that any pinned content will be **available on the public IPFS network**.
+  - Data is **shared**: public IPFS nodes can fetch and host your pinned content. This means your data could be hosted **anywhere** from America, Asia and everything in between. Once the content is delivered to external peers, Scaleway cannot delete it.
+  - Data is **not encrypted**: we **do not apply any encryption** algorithm to your data.
+</Message>
+
+## Scaleway Command Line Interface Overview
+
+[The Scaleway Command Line Interface (CLI)](https://github.com/scaleway/scaleway-cli) allows you to pilot your Scaleway infrastructure directly from your terminal, providing a faster way to administer and monitor your resources. Scaleway CLI is easy to set up and use, and an essential tool for operating efficiently in your cloud environment. It provides many functionalities, including the ability to create and manage volumes and pins.
```
Comment (bene2k1) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:36]:
```suggestion
[The Scaleway Command Line Interface (CLI)](https://github.com/scaleway/scaleway-cli) allows you to pilot your Scaleway infrastructure directly from your terminal, providing a faster way to administer and monitor your resources. Scaleway CLI is an essential tool for operating efficiently in your cloud environment. It provides many functionalities, including the ability to create and manage volumes and pins.
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 4]
```diff
@@ -0,0 +1,172 @@
+---
+meta:
+  title: Getting started with Scaleway IPFS using the CLI
+  description: This page explains how to get started with Scaleway IPFS Pinning using the CLI
+content:
+  h1: Getting started with Scaleway IPFS using CLI
+  paragraph: This page explains how to get started with Scaleway IPFS Pinning using CLI
```
Comment (nerda-codes) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:7]:
```suggestion
  paragraph: This page explains how to get started with Scaleway IPFS Pinning using the CLI
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+</Message>
+
+## Scaleway Command Line Interface overview
+
+[The Scaleway Command Line Interface (CLI)](https://github.com/scaleway/scaleway-cli) allows you to pilot your Scaleway infrastructure directly from your terminal, providing a faster way to administer and monitor your resources. Scaleway CLI is an essential tool for operating efficiently in your cloud environment. It provides many functionalities, including the ability to create and manage volumes and pins.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
```
Comment (nerda-codes) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:41]:
```suggestion
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+  - You have configured your [SSH Key](/console/my-project/how-to/create-ssh-key/)
+  - You have generated your [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+  - You have downloaded and installed the [Scaleway CLI v2](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+  - You have added data to your own local node and retrieved its [CID](/en/docs/labs/ipfs/concepts/#content-identifier-(cid))
```
Comment (nerda-codes) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:45]:
```suggestion
  - You have added data to your own local node and retrieved its [CID](/labs/ipfs/concepts/#content-identifier-(cid))
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+      Region fr-par
+      CountPin 0
+      CreatedAt now
+      UpdatedAt now
+      Name my-volume
+    ```
+
+    | Parameter    | Description                                                                                                                                                                                                                               |
+    | :----------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | `project-id` | ID of the Project you want to create your volume in. Your project name can only contain alphanumeric characters, spaces, dots and dashes. To find your Project ID, you can consult the [Scaleway console](https://console.scaleway.com/)  |
```
Comment (nerda-codes) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:78]:
```suggestion
    | `project-id` | ID of the Project you want to create your volume in. Your Project name can only contain alphanumeric characters, spaces, dots and dashes. To find your Project ID, you can consult the [Scaleway console](https://console.scaleway.com/)  |
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+
+You can delete an entire volume. This will remove all attached pins.
+
+Run the following command to delete your volume:
+
+  ```
+    scw ipfs volume delete volume-id=11111111-1111-1111-1111-111111111111
+  ```
+
+    You should get the following output, confirming your volume has succesfully been deleted.
```
Comment (nerda-codes) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:168]:
```suggestion
    You should get the following output, confirming your volume has been deleted.
```

File: labs/ipfs/quickstart.mdx [hunk 9]
```diff
@@ -17,110 +17,76 @@ categories:
...
 <Macro id="iam-requirements" />
 
 <Message type="requirement">
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
-  - You have configured your [SSH Key](/console/my-project/how-to/create-ssh-key/)
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
   - You have generated your [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
-  - You have downloaded and installed the [Scaleway CLI v2](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+  - You have downloaded and installed [IPFS Desktop App](en/docs/labs/ipfs/reference-content/install-ipfs-desktop)
+  - You have added data to your own local node and retrieved its [CID](/en/docs/labs/ipfs/concepts/#content-identifier-(cid))
```
Comment (nerda-codes) [inline labs/ipfs/quickstart.mdx:40]:
```suggestion
  - You have added data to your own local node and retrieved its [CID](/labs/ipfs/concepts/#content-identifier-(cid))
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+With your content now available on both your local node and our resilient and reliable external nodes, you drastically minimize the risk of disruptions thanks to redundancy. You can also enhance your data storage efficiency by pinning your content to your Scaleway node, and then remove it from your local node.
+
+In this quickstart, we show you how you can [create and manage your volumes](/labs/ipfs/api-cli/volumes-operations) and [pins](/labs/ipfs/api-cli/ipfs-operations) from the [console](https://console.scaleway.com/login), via the [Scaleway Command Line Interface](https://github.com/scaleway/scaleway-cli).
+
+<Message type="important">
+  - Data is **public**: our IPFS nodes are bootstrapped with public IPFS nodes. This implies that any pinned content will be **available on the public IPFS network**.
+  - Data is **shared**: public IPFS nodes can fetch and host your pinned content. This means your data could be hosted **anywhere** from America, Asia and everything in between. Once the content is delivered to external peers, Scaleway cannot delete it.
```
Comment (jcirinosclwy) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:30]:
```suggestion
  - Data is **shared**: public IPFS nodes can fetch and host your pinned content. This means your data could be hosted **anywhere** from America, Asia, and everything in between. Once the content is delivered to external peers, Scaleway cannot delete it.
```

File: labs/ipfs/quickstart.mdx [hunk 11]
```diff
@@ -17,110 +17,76 @@ categories:
...
+
+With your content now available on both your local node and our resilient and reliable external nodes, you drastically minimize the risk of disruptions thanks to redundancy. You can also enhance your data storage efficiency by pinning your content to your Scaleway node, and then remove it from your local node.
+
+In this quickstart, we show you how to add Scaleway as a remote pinning service, create a volume, add a pin to your data and retrieve it using Scaleway IPFS Pinning. 
 
 <Message type="important">
 - Data is **public**: our IPFS nodes are bootstrapped with public IPFS nodes. This implies that any pinned content will be **available on the public IPFS network**.
-- Data is **shared**: public IPFS nodes can fetch and host your pinned content. Your data could be hosted **anywhere** from America, Asia and everything in between. Scaleway is not able to delete it once the content is delivered to external peers.
+- Data is **shared**: public IPFS nodes can fetch and host your pinned content. This means your data could be hosted **anywhere** from America, Asia and everything in between. Once the content is delivered to external peers, Scaleway cannot delete it.
```
Comment (jcirinosclwy) [inline labs/ipfs/quickstart.mdx:30]:
```suggestion
- Data is **shared**: public IPFS nodes can fetch and host your pinned content. This means your data could be hosted **anywhere** from America, Asia, and everything in between. Once the content is delivered to external peers, Scaleway cannot delete it.
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+
+If you no longer wish to store a content on your Scaleway-owned node, you can delete it.
+
+Run the following command to delete your content
+
+  ```
+    scw ipfs pin delete pin-id=0691fe5c-e8ba-40b3-b5cd-bfa69936aac5 volume-id=11111111-1111-1111-1111-111111111111
+  ```
+
+    You should get the following output, confirming your pin has succesfully been deleted on Scaleway's end.
```
Comment (jcirinosclwy) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:152]:
```suggestion
    You should get the following output, confirming your pin has successfully been deleted on Scaleway's end.
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+      Region fr-par
+      CountPin 0
+      CreatedAt now
+      UpdatedAt now
+      Name my-volume
+    ```
+
+    | Parameter    | Description                                                                                                                                                                                                                               |
+    | :----------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | `project-id` | ID of the Project you want to create your volume in. Your Project name can only contain alphanumeric characters, spaces, dots and dashes. To find your Project ID, you can consult the [Scaleway console](https://console.scaleway.com/)  |
```
Comment (jcirinosclwy) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:78]:
```suggestion
    | `project-id` | ID of the Project you want to create your volume in. Your Project name can only contain alphanumeric characters, spaces, dots, and dashes. To find your Project ID, you can consult the [Scaleway console](https://console.scaleway.com/)  |
```

**Review by nerda-codes — APPROVED** [review]

**Review by jcirinosclwy — APPROVED** [review]

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+      Info.Size           40733
+      Info.Progress       100
+      Info.StatusDetails  pinned_ok
+    ```
+
+## How to retrieve pinned content
+
+Once your content is pinned on our service, you can fetch it via any compatible IPFS client connected to the public IPFS network. This can be [Kubo](https://docs.ipfs.tech/install/command-line/), or IPFS desktop.
+
+For the sake of example, we are going to use Kubo. You can find out more about the Kubo CLI on our [dedicated documentation](/labs/ipfs/api-cli/ipfs-cli/)
```
Comment (ldecarvalho-doc) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:125]:
```suggestion
For the sake of this example, we are going to use Kubo. You can find out more about the Kubo CLI on our [dedicated documentation](/labs/ipfs/api-cli/ipfs-cli/)
```

File: labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+      39.78 KiB / 39.78 KiB [============================================] 100.00% 0s
+    ```
+
+2. Navigate your files to retrieve the content. The content is saved under a name that matches its CID.
+
+## How to unpin content
+
+If you no longer wish to store a content on your Scaleway-owned node, you can delete it.
+
+Run the following command to delete your content
```
Comment (ldecarvalho-doc) [inline labs/ipfs/api-cli/getting-started-with-ipfs-cli.mdx:146]:
```suggestion
Run the following command to delete your content:
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**